cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerSetupService.java

```diff
@@ -160,6 +156,30 @@ public void waitForServer() throws CloudbreakException, ClusterClientInitExcepti
         }
     }
 
+    private ApiClient createApiClient() throws ClusterClientInitException {
```

Comment:
There are 2 methods that look the same with minor changes. I'm not sure if this can be moved to some common place or worth generalizing it. It's up to you, just wanted to make a note of it.

Reply:
Their behaviour is quite different. I cannot really come up with a common method for them.